src/pages/system-checks/index.js

```diff
@@ -100,15 +118,28 @@ const SystemChecks = () => {
     })
       .then(() => {})
       .catch(e => {})
-    toast.success('Record Updated Successfully')
+    toast.success(platformLabels.Updated)
   }
 
   return (
     <VertLayout>
+      <Fixed>
+        <Grid container xs={12}>
+          <Grid item xs={3} sx={{ m: 2 }}>
+            <CustomTextField
+              name='search'
+              value={search}
+              label={_labels.search}
+              onClear={() => setSearch('')}
```

Comment:
try to write in the field and then press the clear button ( X ) the value will still the same it will not be cleared